### ISSUE TITLE
Optimising for f2beta_loss

### ISSUE THREAD

**Issue by JoshuaC3** [ZFTurbo/Kaggle-Planet-Understanding-the-Amazon-from-Space#2]
Why didn't you use `f2beta_loss` for the loss function for the CNNs? You used it for the Keras 2nd level model so I wonder why not the others?

Many thanks

**Comment by ZFTurbo** [comment]
It worked a little bit worse for some reason.
